framework/doc/content/source/postprocessors/PseudoTimestep.md

```diff
@@ -0,0 +1,46 @@
+# PseudoTimestep
+
+The methods implemented in this postprocessor compute timesteps adaptively using Pseudo Transient Continuation strategies. This allows for steady state problems solved via time marching approcah to reach the final state rapidly.
+As a reminder the time marching approach to computing steady states reframes a problem in discrete form $M\frac{\partial \mathbf u}{\Delta t}=F(\mathbf u)$ as
+
+\begin{equation}
+M\frac{\mathbf u_{n+1}-\mathbf u_n}{\Delta t}=F(\mathbf u_{n+1})
+\end{equation}
+
+## Overview
+
+This object computes a timestep to accelerate the convergence to steay state using a pseudo transient.
+The change in timestep is determined by the residual behaviour from one iteration to another, i.e. small changes in  residual indicate larger timesteps are allowed, while large changes indicate a decrease is necessary.
+There main methods are recognized and implemented.
```

Comment:
```suggestion
The main methods are recognized and implemented.
```
is this correct?  The wording doesn't seem right.  I would just delete this sentence.

Reply:
Same as above, this is a draft to be extended and run a spell checker before the review process.